ibm_catalog.json

```diff
@@ -0,0 +1,182 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-container-registry",
+      "label": "IBM Container Registry",
```

Comment:
All our DAs have been renamed with a prefix. Can you change this to `Cloud automation for IBM Container Registry`

Reply:
addressed